src/o365/graph/commands/teams/teams-channel-message-list.ts

```diff
@@ -0,0 +1,108 @@
+import auth from '../../GraphAuth';
+import config from '../../../../config';
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption, CommandValidate
+} from '../../../../Command';
+import { GraphItemsListCommand } from '../GraphItemsListCommand';
+import Utils from '../../../../Utils';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  teamId: string;
+  channelId: string;
+}
+
+class GraphTeamsChannelMessageListCommand extends GraphItemsListCommand<any> {
+  public get name(): string {
+    return `${commands.TEAMS_CHANNEL_MESSAGE_LIST}`;
+  }
+
+  public get description(): string {
+    return 'Lists all the messages from a channel in a Microsoft Teams team';
+  }
+
+  public commandAction(cmd: CommandInstance, args: CommandArgs, cb: () => void): void {
+    let endpoint: string = `${auth.service.resource}/beta/teams/${args.options.teamId}/channels/${args.options.channelId}/messages`;
```

Comment:
Since the endpoint is not changing it should be a `const` instead of `let`

Reply:
Agree.